### PR TITLE
The Rollable Cigs and Chews Update

### PR DESCRIPTION
**Pull request by alsoandanswer**
Adds rolling tobacco and chewing tobacco, and all the bells and whistles that come along with it.
Also adds new sprites for cigs and lighters in general. (tweaked eris sprites, should look visually distinct, anything past cigar packets is completely my design)

dreammaker_mx6p64pkc7

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by Geevies** [comment]
The closed cigar case leaves a bit to be desired. Can it be given a larger bevel or some decals or something?

**Comment by alsoandanswer** [comment]
alright i tweaked it to be more shiny
dreammaker_SpFCGbqDFn

**Comment by KingOfThePing** [comment]
ngl you're doing god's work son

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -244,21 +252,18 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
 		var/obj/item/blocked = H.check_mouth_coverage()
 		if(blocked)
-			to_chat(H, "<span class='warning'>\The [blocked] is in the way!</span>")
+			to_chat(H, span("warning", "\The [blocked] is in the way!"))
 			return 1
-		if(last_drag <= world.time - 30) //Spam limiter. Only for messages/sound.
-			last_drag = world.time
-			H.visible_message("<span class='notice'>[H.name] takes a drag of their [name].</span>")
-			playsound(H, 'sound/items/cigs_lighters/inhale.ogg', 50, 0, -1)
-		reagents.trans_to_mob(H, (rand(10,20)/10), CHEM_BREATHE) //Smokes it faster. Slightly random amount.
```
Comment (fernerr) [inline code/game/objects/items/weapons/cigs_lighters.dm]: Was this mistakenly removed in the merge perhaps?

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**